samwise.js

```diff
@@ -5,7 +5,6 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?